src/holon/rule_engine/scenario_aggregate.py

```diff
@@ -75,7 +77,7 @@ def add_object(
         for key, value in additional_attributes.items():
             setattr(object, key, value)
 
-        self.repositories[base_model_type].add(object)
+        return self.repositories[base_model_type].add(object)
```

Comment:
De add gebruiken we in de rule actions, waar we alleen items direct aan de repo toevoegen van de ScenarioAggregate. Een object returnen willen we juist vermijden, omdat we dan niet zeker weten of het ge-returnde object nog steeds correct gelinkt is aan het SA right?

Reply:
Het gegenereerde id en het object zelf dat is toegevoegd,is nodig, omdat andere nieuwe objecten daar naar moeten refereren. Aangezien daar in de code relatie velden en relatie id velden worden geüpdatet, gaat dat goed.

Reply:
Ah check, ik had het niet goed gelezen. Het returnt het nieuw toegevoegde object, niet de geupdate repository. Ik heb niks gezegd